google-cloud-storage/lib/google/cloud/storage.rb

```diff
@@ -317,6 +317,23 @@ module Cloud
     # downloaded.read #=> "Hello world!"
     # ```
     #
+    # Download a public file with an anonymous, unauthenticated client. Use
+    # `skip_lookup` to avoid errors retrieving non-public bucket and file
+    # metadata.
```

Comment:
> Use `skip_lookup` to avoid errors retrieving non-public bucket and file metadata.
  
@jskeet This advice is appropriate, isn't it?

Reply:
I don't know what skip_lookup does... I don't think we have any similar concept in the C# code. (When we fetch the data in C#, we don't fetch metadata, and vice versa.)